src/IECoreDelight/ParameterList.cpp

```diff
@@ -143,6 +153,11 @@ NSIParam_t ParameterList::parameter( const char *name, const IECore::Data *value
 		0  // flags
 	};
 
+	if( value->typeId() == V2fVectorDataTypeId && isSingleArray )
+	{
+		msg( Msg::Warning, "ParameterList", fmt::format( "Attribute \"{}\" cannot be converted to an array of V2f parameters.", name ) );
+	}
```

Comment:
Could we move this into the `case V2fVectorDataTypeId` block? It would explain in-situ why we treat it differently to the other array types. Up here there's not much association between the two.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5432/commits/efa81ca66e92e0d046f23bacf1e233cbfeb98067